monai/data/wsi_datasets.py

```diff
@@ -171,26 +171,29 @@ def _transform(self, index: int):
 
 class SlidingPatchWSIDataset(Randomizable, PatchWSIDataset):
     """
-    This dataset extracts patches from whole slide images (without loading the whole image)
+    This dataset extracts patches in sliding-window manner from whole slide images (without loading the whole image).
     It also reads labels for each patch and provides each patch with its associated class labels.
 
     Args:
         data: the list of input samples including image, location, and label (see the note below for more details).
-        size: the size of patch to be extracted from the whole slide image.
-        level: the level at which the patches to be extracted (default to 0).
+        patch_size: the size of patch to be extracted from the whole slide image.
```

Comment:
the same issue is in the other modules such as `PatchWSIDataset`, please help review the other wsi related modules as you did a few rounds of refactoring earlier.

Reply:
Thanks for pointing this out. Will fix it now.

Reply:
done